sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/implementation/util/ImplUtils.java

```diff
@@ -180,120 +162,6 @@ public static byte[] byteBufferToArray(ByteBuffer byteBuffer) {
         return byteArray;
     }
 
-    /**
-     * Utility method for parsing a {@link URL} into a {@link UrlBuilder}.
-     *
-     * @param url The URL being parsed.
-     * @param includeQuery Whether the query string should be excluded.
-     * @return The UrlBuilder that represents the parsed URL.
-     */
-    public static UrlBuilder parseUrl(URL url, boolean includeQuery) {
```

Comment:
Are these two utility methods no longer needed?

Reply:
yeah, they were unused in the current state of repo. We can always add back if needed.